repos/system_upgrade/el7toel8/actors/bootentriesscanner/libraries/bootentriesscanner.py

```diff
@@ -0,0 +1,47 @@
+from leapp.models import BootEntry
+from leapp.libraries.stdlib import run, api
+
+
+CMD_GRUBBY_INFO_ALL = ['grubby', '--info', 'ALL']
+
+
+def remove_quotes_if_used(text):
+    """
+    Removes the first pair of quotes surrounding the given text if any.
+
+    :param text: str
+    :rtype: str
+    :return: The given text with the first pair of quotes removed if present.
+    """
+
+    if len(text) < 2:
+        return text
+
+    if text[0] == text[-1] and text[0] in ['\'', '\"']:
+        return text[1:-1]
+    return text
+
+
+def scan_boot_entries():
+    """
+    Scans the available boot entries are informs other actors about the findings
+    via a :class:`BootEntry` message.
+    """
+    grubby_output = run(CMD_GRUBBY_INFO_ALL, split=True)
+
+    boot_entries = []
+    # Identify the available boot entries by searching for their titles in the grubby output
+    for output_line in grubby_output['stdout']:
+        # For now it is sufficient to look only for the titles as that is the only
+        # information we use. If need be, we would have to parse the structure
+        # of the grubby output into sections according to the `index` lines
+        if output_line.startswith('title='):
+            boot_entry = output_line[6:]  # Remove the `title=` prefix
+
+            # On s390x grubby produces quotes only when needed (whitespace in
+            # the configuration values), on x86 the values are quoted either way
+            boot_entry = remove_quotes_if_used(boot_entry)
```

Comment:
```suggestion
            boot_entry = boot_entry.strip('\'"')
```

Reply:
I don't think that you want any quotes around it

Reply:
TIL: U can specify characters to be stripped via an optional argument. 
Awesome! Thank you.

Reply:
Fixed in the latest commit.